opal_in_production.org

```diff
@@ -84,18 +84,18 @@ Some Key Points:
     sprockets to load assets, one CAN setup a situation where you
     wouldn't need to require things.  We did encounter load problems,
     which in part are due to loading specs in browser that were
-    troublesome.  Instead we went with explicit requires in our Opal
+    troublesome.  Instead we fell back to using explicit requires in our Opal
```

Comment:
I think it's good to acknowledge that not everything is perfect w/o moping about it.  btw, your rewording solves my issue  in my previous PR